tests/shared/src/test/scala/scala/meta/tests/parsers/ScaladocParserSuite.scala

```diff
@@ -452,25 +480,33 @@ class ScaladocParserSuite extends FunSuite {
           */
          """
       )
-    val expected = Option(
-      Scaladoc(
-        Seq(
-          Paragraph(
-            Seq(
-              Text(Seq(Word("Some"), Word("text:"))),
-              ListBlock(
-                "-",
-                Seq(
-                  ListItem(
-                    Text(Seq(Word(list11))),
-                    Some(ListBlock("1.", Seq(ListItem(Text(Seq(Word(list21)))))))
-                  ),
-                  ListItem(
-                    Text(Seq(Word(list12))),
-                    Some(
-                      ListBlock(
-                        "a.",
-                        Seq(ListItem(Text(Seq(Word(list21)))), ListItem(Text(Seq(Word(list22)))))
+
+    val expected =
```

Comment:
most of this diff contains formatting changes (newline before `Option`) and type `Seq -> List`. i'd keep just the actual changed part.